### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by chimwemwe007**
# Bookstore: styling
### I did the following.

- Use the information detailed in Zeplin to style the website.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,27 +1,321 @@
...
+
+.book {
+  background-color: white;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  padding: 2rem;
+}
+
+.book-info {
+  display: flex;
```
Comment (Davidosky007) [inline src/App.css:75]: * Kindly dont repeat the same style rule as it is a [CSS bad practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)  as i can see you keep repeating styles like `display flex` `margin` align items centre` etc [Go through this article on how to properly style in CSS](https://speckyboy.com/good-bad-css-practices/)

File: src/App.css [hunk 2]
```diff
@@ -1,27 +1,321 @@
...
+div[role="progressbar"] {
+  --size: 4.24rem;
+  --fg: #307bbe;
+  --bg: #e8e8e8;
+  --pgPercentage: var(--value);
+
+  animation: growProgressBar 3s 1 forwards;
+  width: var(--size);
+  height: var(--size);
+  border-radius: 50%;
```
Comment (Davidosky007) [inline src/App.css:249]: * As stated in the project requirement `Your final design should match Zeplin's design.` currently, your design doesn't match the given design below are screenshots of highlighted areas in the design you should work on ( which are colours, margins, font sizes, and missing contents)
image

## YOURS 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @chimwemwe007 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Proper use of Gitflow 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Davidosky007** [comment]
Kindly submit for review through your dashboard

File: src/App.js [hunk 3]
```diff
@@ -1,5 +1,7 @@
 import './App.css';
 import { Routes, Route } from 'react-router-dom';
```
Comment (addod19) [inline src/App.js:2]: - [ ] I'm unable to install packages or run and test the app. Here is a screenshot of the error I'm getting on my system
Screenshot from 2022-08-27 13-07-13

**Review by addod19 — CHANGES_REQUESTED** [review]
Hi @chimwemwe007, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [ ] Nice job implementing the changes, I can see some file changes and commits :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 3 above)
Comment (chimwemwe007) [reply src/App.js:2]: alright let me check since it was running properlyon mine

Comment (addod19) [reply src/App.js:2]: Kindly do that and if possible add a screenshot of the UI

Comment (chimwemwe007) [reply src/App.js:2]: try deleting the package.lock.jason and running it again

**Review by BertrandConxy — APPROVED** [review]
Hi @chimwemwe007 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 

- No linter errors
- project requirements are met
- Good UI
- project works fine
- correct git-flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**